ProjectSidewalk/SidewalkWebpage
# #2502: Start auditing a street from label map

Almost complete.

Added a link to an expired image window that allows the user to be redirected to the label to reaudit. Currently redirects to the correct street. Lat, lng, and panoId need to be added in the route parameters.

##### Testing instructions
1. Open the label map and click a label that is expired
2. Click the link that suggests the user should 'explore the street again'
